src/optick.h

```diff
@@ -83,7 +83,7 @@
 #endif
 
 // Vulkan Forward Declarations
-#ifndef OPTICK_ENABLE_GPU_VULKAN
+#if OPTICK_ENABLE_GPU_VULKAN == 0

```

Comment:
This change requires OPTICK_ENABLE_GPU=0 define for all the projects which don't need vulkan. I'd rather keep "Opt In" behaviour by default for Vulkan, than "Opt Out".

Reply:
I think that revision is from an outdated revision. Looking at it now I may have messed up my pull request some how. Seems like some old changes that already got merged ended up in this PR? Not sure how that happened.